solr/solr-ref-guide/modules/getting-started/pages/tutorial-films.adoc

```diff
@@ -340,13 +352,13 @@ If we construct a query that looks like this:
 
 [source,bash]
 curl 'http://localhost:8983/solr/films/select?q=*:*&rows=0'\
-    '&facet=true'\
-    '&facet.range=initial_release_date'\
-    '&facet.range.start=NOW-20YEAR'\
-    '&facet.range.end=NOW'\
-    '&facet.range.gap=%2B1YEAR'
+'&facet=true'\
+'&facet.range=initial_release_date'\
+'&facet.range.start=NOW/YEAR-25YEAR'\
+'&facet.range.end=NOW'\
+'&facet.range.gap=%2B1YEAR'
 
-This will request all films and ask for them to be grouped by year starting with 20 years ago (our earliest release date is in 2000) and ending today.
+This will request all films and ask for them to be grouped by year starting with 25 years ago (our earliest release date is in 2000) and ending today.
```

Comment:
25 years should be 1997. Any specific reason you switched this to 25 ?

Reply:
Because 20 years was no longer enough to cover the whole range (as it was when the tutorial was originally written, guessing 2015 or so)